package.json

```
@@ -14,19 +14,21 @@
   "main": "index.js",
   "source": true,
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "eslint --ignore-path .gitignore ."
   },
   "dependencies": {},
   "devDependencies": {
     "@babel/core": "7.7.4",
-    "@babel/register": "7.7.4",
-    "@babel/preset-env": "7.7.4",
+    "@babel/plugin-proposal-class-properties": "7.7.4",
     "@babel/plugin-proposal-function-bind": "7.7.4",
     "@babel/plugin-syntax-object-rest-spread": "7.7.4",
-    "@babel/plugin-proposal-class-properties": "7.7.4",
+    "@babel/preset-env": "7.7.4",
+    "@babel/register": "7.7.4",
     "bufferutil": "^4.0.1",
     "canvas": "^2.5.0",
     "chai": "4.2.0",
+    "eslint": "^6.8.0",
```

```suggestion
    "eslint": "6.8.0",
```

@ChevonNwosu can you please remove the ^carrot from all these? We want to have locked versions.